### DISCUSSION TITLE
How to disable some typecheck rules?

### DISCUSSION THREAD

**Discussion by hoangitk** [vuedx/languagetools#156] [Q&A] [answered]
Hi,

Is there a way to disable some rules like VueDX/TS(2604)?

![image](https://user-images.githubusercontent.com/342078/104563998-f61d9780-567c-11eb-8e89-fbecc86ddfbd.png)

Thank you!

**Comment by znck** [comment]
There's nothing at the moment but I can try adding template equivalent of `@ts-ignore`.

**Reply by murphybob** [reply]
Related to this, is there currently a way to disable typecheck rules in the script portion of an SFC?  I was playing with script setup components (I know it's still preview, just interested because the syntax is 🤌) but adding @ts-ignore above declarations still left me with 'declared but never used' warnings

![image](https://user-images.githubusercontent.com/1550045/105826447-a57d3700-5fb8-11eb-8dd2-2206f7b484a2.png)

p.s. loving this project 💯

**Reply by znck** [reply]
That looks like a bug, script block should behave like any other JS/TS file. Please create an issue for this.

**Comment by znck** [comment, marked as answer]
Tracked as https://github.com/vuedx/languagetools/issues/285
